servicex/yaml_parser/parser.py

```diff
@@ -0,0 +1,195 @@
+# Original code released under
+# The MIT License (MIT)
```

Comment:
An this is smart @ponyisi. I didn't even think about just adding the license to the code itself. :+1: